requirements.txt

```diff
@@ -10,5 +10,8 @@ openai>=1.13.3,<2.0.0
 #       removed once that one is removed.
 # do not use 8.4.0 due to a bug in the library
 # https://github.com/instructlab/instructlab/issues/1389
+pypdf>=5.0.0
+tabulate>=0.9.0
 tenacity>=8.3.0,!=8.4.0
+transformers>=4.44.2
```

Comment:
Should we align this with instructlab/instructlab? Currently that's on 4.41.2 at https://github.com/instructlab/instructlab/blob/main/requirements.txt#L34